test/functional/apps/dashboard/dashboard_filtering.js

```diff
@@ -72,6 +79,10 @@ export default function ({ getService, getPageObjects }) {
 
     describe('adding a filter that excludes all data', () => {
       before(async () => {
+        await PageObjects.common.navigateToApp('discover');
+        await PageObjects.discover.switchDiscoverTable('new');
+        await PageObjects.common.navigateToApp('dashboard');
+        await PageObjects.dashboard.gotoDashboardLandingPage();
```

Comment:
We need to go to the landing page?

Reply:
yea need to go to landing page first for clickNewDashboard() to work